man/coap-client.txt.in

```diff
@@ -41,7 +41,10 @@ IPv6 address references, angle brackets are required (c.f. EXAMPLES).
 OPTIONS - General
 -----------------
 *-a* addr::
-   The local address of the interface that has to be used.
+   The local address of the interface that has to be used. +
```

Comment:
I think, the `+` at the end of this line is a copy&paste artifact and can be removed?

Reply:
The plus forces the text to be wrapped to the next line (the Note) but still indented, so should be there.

Reply:
My fault, I am not that familiar with asciidoc.